data/json/items/tool/electronics.json

```
@@ -424,6 +424,79 @@
     ],
     "tick_action": [ "EPIC_MUSIC" ],
     "flags": [ "WATCH", "LIGHT_10", "TRADER_AVOID", "WATER_BREAK", "ELECTRONIC", "ALLOWS_REMOTE_USE" ]
+},
+  {
+    "id": "portable_dvd_player",
+    "type": "TOOL",
+    "category": "tools",
+    "name": { "str": "portable DVD player" },
+    "description": "A compact device with a 7-inch screen designed for playing both CDs and DVDs.  It's portable, making it convenient for travel or on-the-go entertainment in the Cataclysm.  This device also include simple earphones, built-in speakers, a rechargeable battery, and an USB and SD card slots for multimedia playback."
```

Pardon me while I head off to recover from the Mack truck of nostalgia this PR slammed into me at lethal speeds.
```suggestion
    "description": "A compact device with a 7-inch screen, designed for playing both CDs and DVDs.  It's portable, making it convenient for travel or on-the-go entertainment in the Cataclysm.  This device also includes simple earphones, built-in speakers, a rechargeable battery, and USB and SD card slots for multimedia playback."
```